packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -462,26 +493,148 @@ class CupertinoPopupSurface extends StatelessWidget {
   /// Some popups, like iOS's volume control popup, choose to render a blurred
   /// area without any white paint covering it. To achieve this effect,
   /// [isSurfacePainted] should be set to false.
+  ///
+  /// Defaults to true.
   final bool isSurfacePainted;
 
   /// The widget below this widget in the tree.
-  final Widget? child;
+  ///
+  /// Because [CupertinoPopupSurface] is composed of proxy boxes, which mimic
+  /// the size of their child, a [child] is required to ensure that this surface
+  /// has a size.
+  final Widget child;
+
+  /// The default strength of the blur applied to widgets underlying a
+  /// [CupertinoPopupSurface].
+  ///
+  /// Setting this to 0 will remove the blur filter.
+  static const double defaultBlurSigma = 30.0;
```

Comment:
While I appreciate the design of making default values a variable, the duplicate `30.0` is concerning. Maybe:
```suggestion
  static const double defaultBlurSigma = _kBlurAmount;
```

Reply:
Done, although I should note that _CupertinoActionSheetState used _kBlurAmount, which I replaced with CupertinoPopupSurface.defaultBlurSigma.

Reply:
Oh that's interesting! I wonder why it doesn't use `CupertinoPopupSurface`. It probably should.